src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -109,11 +109,10 @@ function renderTooltip(description: string) {
   );
 }
 
-const VISUALIZE_EMBEDDABLE_TYPE = 'visualization';
 type VisualizeEmbeddable = any;
 
 function getViewDescription(embeddable: IEmbeddable | VisualizeEmbeddable) {
-  if (embeddable.type === VISUALIZE_EMBEDDABLE_TYPE) {
+  if (embeddable.getVisualizationDescription) {
```

Comment:
Could this be `embeddable.getDescription` instead? Not all embeddables are visualizations